charts/brigade/templates/artemis/primary-stateful-set.yaml

```diff
@@ -88,6 +88,11 @@ spec:
             port: core
           initialDelaySeconds: 10
           periodSeconds: 10
+        startupProbe:
+          tcpSocket:
+            port: core
+          failureThreshold: 10
+          periodSeconds: 10
```

Comment:
Same comment as [here](https://github.com/brigadecore/brigade/pull/1304/files#r585681169) though perhaps even more relevant in relation to Artemis?

Reply:
The Artemis stateful-set yamls indeed already came with liveness and readiness probes; just added startupProbes to each in this PR.

Reply:
Sorry... the comment here was really about needing a higher failure threshold.